doc/howto.rst

```diff
@@ -95,7 +98,15 @@ i915 Linux perf data. Follow the above BKM to run it.
 
 For further reading on the Linux perf security refer to Linux kernel
 `perf-security <https://www.kernel.org/doc/html/latest/admin-guide/perf-security.html>`_
-admin guide.
+admin guide and ``man capabilities``.
+
+With latest container manager and runtime versions (podman, containerd, runc),
+CAP_PERFMON (added in kernel v5.8) can be used instead of the (too wide) CAP_SYS_ADMIN
+capability. However, at the moment Docker does not support CAP_PERFMON:
+
+* https://github.com/docker/docs/pull/15050 - CAP_PERFMON removed from release notes
+* https://github.com/moby/moby/pull/42011 - CAP_PERFMON support reverted
+* https://github.com/docker/docs/issues/14124 - original issue reported against CAP_PERFMON support in docker
```

Comment:
@eero-t : I did above changes. Please, take a look. If they look good to you, I will merge.

Reply:
Looks good to me!